.travis.yml

```diff
@@ -76,24 +76,24 @@ before_script:
     - gem install sass
     - gem install compass
     - npm install -g npm
-    - npm install -g grunt-cli
+    - npm install -g gulp-cli
     - npm install
 
-script: grunt $WP_TRAVISCI
+script: gulp $WP_TRAVISCI
 
 sudo: false
 
 notifications:
    email:
-       - enej.bajgoric@automattic.com
-       - georgestephanis@automattic.com
-       - jeremy@automattic.com
-       - miguel@automattic.com
-       - rocco@automattic.com
-       - smart@automattic.com
-       - eric.binnion@automattic.com
-       - allendav@automattic.com
-       - beau@automattic.com
-       - kraft@automattic.com
+#       - enej.bajgoric@automattic.com
```

Comment:
I think you'll want to uncomment our email addresses before to merge. I like my Travis emails :)